### PR TITLE
feat: add ree_pipeline to academy

### PR DESCRIPTION
**Pull request by ubabe53**
I have followed the same flow of Fonseca's pipeline. As I mentioned before it was tailored for that use case, I used another dataset, I specified things in the README

### CODE REVIEW

File: missing-values-imputer/wind-imputation/preprocess.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+from ydata.dataset import Dataset
+
+def preprocess_data(df: Dataset):
```
Comment (jfsantos-ds) [inline missing-values-imputer/wind-imputation/preprocess.py:3]: This definition also appears in the Preprocess notebook. Should we keep it in only one place and import it in other instances?

File: missing-values-imputer/wind-imputation/utils.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from ydata.dataset import Dataset
+
+
+def preprocess_data(data: Dataset):
```
Comment (jfsantos-ds) [inline missing-values-imputer/wind-imputation/utils.py:4]: Shared name with the other preprocess method. Should this one be named more in the line of  _setting_ and _sorting_ index?

File: missing-values-imputer/wind-imputation/preprocess.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    df = df.select_columns(list(DTYPES.keys()))
+
+    # Convert to pandas and cast to correct data types
+    df = df.to_pandas().astype(DTYPES)
+    
+    # Drop duplicates
+    df = df.drop_duplicates()
+
+    # Sort and set index, sort columns
+    df = df.sort_values(by=['timestamp', 'station']).set_index('timestamp')
```
Comment (jfsantos-ds) [inline missing-values-imputer/wind-imputation/preprocess.py:24]: It seems that we could call the method found in the `missing-values-imputer/wind-imputation/utils.py` directly in this method. Wdyt?

File: missing-values-imputer/wind-imputation/3_Imputation_ColdStart.ipynb [hunk 4]
```diff
@@ -0,0 +1,770 @@
...
+      "\u001b[0;32m/opt/conda/lib/python3.7/site-packages/distributed/comm/core.py\u001b[0m in \u001b[0;36mconnect\u001b[0;34m(addr, timeout, deserialize, handshake_overrides, **connection_args)\u001b[0m\n\u001b[1;32m    315\u001b[0m         raise OSError(\n\u001b[1;32m    316\u001b[0m             \u001b[0;34mf\"Timed out trying to connect to {addr} after {timeout} s\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 317\u001b[0;31m         ) from active_exception\n\u001b[0m\u001b[1;32m    318\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    319\u001b[0m     local_info = {\n",
+      "\u001b[0;31mOSError\u001b[0m: Timed out trying to connect to gateway://gateway.dask.svc.cluster.local:80/dask.b8671cbf08b747008d39d02c4dff82f1 after 30 s"
```
Comment (jfsantos-ds) [inline missing-values-imputer/wind-imputation/3_Imputation_ColdStart.ipynb:715]: Is this a recurrent error?

**Review by jfsantos-ds — APPROVED** [review]
Seems good to me.

Left some comments here and there, and have a few general comments:
* The files added on this PR seem like they could be nested in a less specific directory. Instead of `missing-values_imputer/` I'd suggest something like `data-quality/missing-values-imputation/`. Since @gmartinsribeiro created a directory for data quality inspection he might have a more wholesome suggestion regarding data quality
* Some notebook write file cells will do better with suppressed output. Example, "3 - Store Data" cell in [Preprocess notebook](academy/missing-values-imputer/wind-imputation/1_Preprocess.ipynb). If this seems to be off the point (because the notebooks will be run in a pipeline instead of statically stored), I suggest trying the `%%capture` command in the beginning of each cell that needs output suppression
* The error in writing file, in the cold start imputation, should be fixed so the notebook can be reuploaded

File: missing-values-imputer/wind-imputation/preprocess.py (hunk 1 above)
Comment (ubabe53) [reply missing-values-imputer/wind-imputation/preprocess.py:3]: You saw the other one, they're slightly different , I'm going to rename one of them

File: missing-values-imputer/wind-imputation/utils.py (hunk 2 above)
Comment (ubabe53) [reply missing-values-imputer/wind-imputation/utils.py:4]: Yep, I'm going to rename it

**Review by fabclmnt — APPROVED** [review]

**Pull request merged**